estimator/estimator.go

```diff
@@ -154,7 +154,7 @@ func (e *Estimator) getAvgBlockDurationNanos() (time.Duration, error) {
 
 func (e *Estimator) CalcBlock(ttime time.Time) (int64, error) {
 	if ttime.Unix() <= time.Now().Unix() {
-		return 0, fmt.Errorf("time to estimate is < less than the current time")
+		return 0, fmt.Errorf("date to estimate must be in the future")
```

Comment:
should that be "height to estimate ..."?

the user doesn't give a date as input, do they?